### PR TITLE
BlizzardEntertainment.Battle.netSetup version 1.18.5.3106

### PR DESCRIPTION
**Pull request by PotentCricket**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/100547)

### CODE REVIEW

File: manifests/b/BlizzardEntertainment/Battle/netSetup/1.18.5.3106/BlizzardEntertainment.Battle.netSetup.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: BlizzardEntertainment.Battle.netSetup
+PackageVersion: 1.18.5.3106
+Installers:
+- InstallerUrl: https://us.battle.net/download/getInstaller?os=win&installer=Battle.net-Setup.exe&id=undefined
```
Comment (rodrigoborgesdeoliveira) [inline manifests/b/BlizzardEntertainment/Battle/netSetup/1.18.5.3106/BlizzardEntertainment.Battle.netSetup.installer.yaml:7]: This file return a different hash every time it is downloaded.

Comment (PotentCricket) [reply manifests/b/BlizzardEntertainment/Battle/netSetup/1.18.5.3106/BlizzardEntertainment.Battle.netSetup.installer.yaml:7]: Yeah I just figured that out. Apologies, this was my first attempt at winget using wingetcreate. I have closed the request, as I'm unsure how to fix this for now.

Comment (rodrigoborgesdeoliveira) [reply manifests/b/BlizzardEntertainment/Battle/netSetup/1.18.5.3106/BlizzardEntertainment.Battle.netSetup.installer.yaml:7]: No worries! Thanks for submitting your first PR. If you can find a direct link, feel free to open a new PR/reopen this one. For Battlenet specifically, it has had issues in the past as well:
- https://github.com/microsoft/winget-pkgs/pull/60225#pullrequestreview-971013280

Comment (PotentCricket) [reply manifests/b/BlizzardEntertainment/Battle/netSetup/1.18.5.3106/BlizzardEntertainment.Battle.netSetup.installer.yaml:7]: oh i see! I just read about the CDN and region stuff there. I did search beforehand but nothing for blizzard showed up, I forgot to check closed requests lol. Thanks for the info :)

**Pull request closed without merging**